scripts/build-js

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
```

Comment:
this looks weird.

Why the `env`? Why `bash` and not just 'sh'?
```suggestion
#!/bin/sh
```

Reply:
`bash` is a lot more capable and a syntax we're more familiar with. Even if we're not using any features that aren't POSIX compliant here, simply the fact that we're more used to writing `bash` (and that it supports nicer syntax) seems like enough of a reason to use it to me. `bash` is available in the contexts we care about - Linux & MacOS - and generally we run this through the `./run` script anyway which uses our development docker container.

The `/usr/bin/env` question - basically we do this for consistency with how we invoke Python, which we often use as an interpreter in scripts instead. The rationale is really the same - you may for some reason want to override or fake the environment in which you're running a project, and we'd like to support other workflows - although I accept that in practice this is not common with `bash`. But generally it seems a little safer to interrogate the environment to find the binary rather than assume it's at `/bin/bash`. [Some advice](https://unix.stackexchange.com/a/77586/10039) out there suggests that it's generally better to run anything except `/bin/sh` with `env`. Or maybe in the future we'll replace our development tooling with something that encapsulates binaries in a directory. Either way, `/usr/bin/env` doesn't seem to do any harm and may be adding flexibility. Why, what harm do you think it does?